docs/topics/forms/required/mark-non-required-fields.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Mark non-required fields
+layout: default
+parent: Required fields
+nav_order: 3
+---
+
+# Option 2: Marking only the non-required form fields
```

Comment:
Omit "the": "Marking only non-required form fields"

Question: do we want to say "non-required" or "optional"? I feel that optional is clearer and easier to read. Having the variants be "required" and "non-required" means that you have to read more carefully to be sure whether you're looking at the right instructions. This is something consistent through these docs.

Reply:
agree